homeassistant/util/logging.py

```diff
@@ -53,7 +58,7 @@ def handle(self, record: logging.LogRecord) -> None:
             self._reset_counters(record.created)
 
         module_name = record.name
-        if module_name == __name__ or module_name in self._warned_modules:
+        if self._check_skip_flag(module_name):
```

Comment:
I suggest we avoid the function call here by only calling if `module_name` is not in the dict